src/convertToObject.js

```diff
@@ -6,7 +6,26 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
-}
+  let strToObject = sourceString
+    .trim()
+    .split(';')
+    .filter((str) => str.trim());
+
+  strToObject = strToObject.map((str) => {
```

Comment:
The 'map' method should be chained directly after the 'filter' method for better readability, following the indentation style of chained methods.